LSPS6/README.md

```diff
@@ -0,0 +1,130 @@
+# LSPS6
+
+| Name    | `.well-known`      |
+| ------- |--------------------|
+| Version | 1                  |
+| Status  | For Implementation |
+
+## Motivation
+
+The purpose is to provide a standard way for clients to obtain extra information from the LSP.
+There might be information that is not available through regular LSPS messages.
+An example for such information could be the pricing strategy for LSPS1.
+Optionally there is also a mechanism to show authoritatively that the information
+is coming from the LSP in order to prevent spoofing.
+
+### Actors
+
+The 'LSP' is the API provider, and acts as the server.
+The 'client' is the API consumer.
+
+## Well-Known URI
+
+[Well-known URIs][] are a way to discover information about a service.
+The LSP SHOULD serve a well-known URI `/.well-known/bitcoin` to allow for public key discovery.
+
+```json
+{
+  "icon": "https://acme.com/icon.png",
+  "logo": "https://acme.com/logo.png",
+  "email": "support@acme.com",
+  "nostr": "npub1w...s2mfp9",
+  "twitter": "@acme",
+  "telegram": "@acme",
+  "lightning": {
+    "mainnet": {
+      "public_keys": [
+        "0326e6...2833a3",
+        "0305f5...2e4e45"
+      ]
+    },
+    "testnet": {
+      "public_keys": [
+        "03f060...869c00"
+      ]
+    },
+    ...
+  }
+}
+```
+
+- `icon` LSP MAY include the URL of its icon.
+- `logo` LSP MAY include the URL of its logo.
+- `email` LSP MAY include its support email.
+- `nostr` LSP MAY include its nost public key.
+- `twitter` LSP MAY include its Twitter handle.
+- `telegram` LSP MAY include its Telegram handle.
+- `lightning`
+  - `mainnet` LSP SHOULD include its mainnet lightning node public keys.
+  - `testnet` LSP MAY include its testnet lightning node public keys.
+  - `...` LSP MAY include its public keys for other network types.
+
+[Well-known URIs]: https://datatracker.ietf.org/doc/html/rfc8615
+
+## API
+
+### lsps6.get_info
+
+`lsps6.get_info` provides generic information about the LSP.
+
+The client SHOULD use the data in `url` up until the [TLD][] and append `/.well-known/bitcoin`.
+So assuming the URL is `https://acme.com/lsp`, the client MUST use `https://acme.com/.well-known/bitcoin`.
+To confirm authority the client MUST crawl this URL and
+match the public key from the LSP with the public key from the well-known URI.
+
+[TLD]: https://en.wikipedia.org/wiki/Top-level_domain
+
+Any information that is available in the response of `lsps6.get_info`
+should override the information in the well-known URI as it's more authoritative.
+
+If there is a piece of information unavailable,
+the client SHOULD use the information from the well-known URI as a fallback.
+
+**Request** No parameters needed.
+
+**Response**
+
+```JSON
+{
+  "url": "https://acme.com/lsp",
+  "icon": "https://acme.com/lsp-icon.png",
+  "logo": "https://acme.com/lsp-logo.png",
```

Comment:
What's the difference between icon and logo? Does it need to be in a specific format? Maybe a specific resolution?